cern_access/indico_cern_access/models/access_requests.py

```diff
@@ -40,16 +40,16 @@ class CERNAccessRequest(db.Model):
         nullable=False
     )
     birth_date = db.Column(
-        db.Date,
-        nullable=True
+        db.Date
     )
     nationality = db.Column(
-        db.String,
-        nullable=True
+        db.String
     )
     birth_place = db.Column(
-        db.String,
-        nullable=True
+        db.String
+    )
+    license_plate = db.Column(
```

Comment:
I think `clear_identity_data` below should clear the license plate as well

Reply:
Indeed, thanks!